src/operator/src/statement/copy_table_from.rs

```diff
@@ -379,11 +378,23 @@ impl StatementExecutor {
 
         let mut rows_inserted = 0;
         let mut insert_cost = 0;
-        let max_insert_rows = req
-            .with
-            .get(MAX_INSERT_ROWS)
-            .and_then(|val| val.parse::<usize>().ok())
-            .unwrap_or(DEFAULT_MAX_INSERT_ROWS);
+        let max_insert_rows = if let Some(num) = req.limit {
+            Some(num as usize)
+        } else {
+            match req.with.get(MAX_INSERT_ROWS) {
+                Some(num) => match num.parse::<usize>() {
+                    Ok(num) => Some(num),
+                    Err(_) => {
+                        return Err(error::InvalidCopyParameterSnafu {
```

Comment:
nitpick: Maybe we could add a sqlness test that the max_insert_rows option is invalid.

Reply:
I see we also support the following syntax:
```
Copy with_limit_rows FROM '/tmp/demo/export/parquet_files/' WITH (MAX_INSERT_ROWS = 2);
```

But if we already support the limit syntax, I don't see the necessity to provide two ways to set this limit.